src/SourceIndex/HtmlGenerator.exe.config

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If we want to fork and maintain that nupkg then this can be placed in there. It is in here because we need to add a binding redirect for Microsoft.Build.Framework to make loading our tasks work.